### PR TITLE
Issue 88: Update HiDPI units and `hidpi-bg-img` @mixin

### LINKED ISSUE #88: Update HiDPI units and `hidpi-bg-img` @mixin
See latest [Scally version](https://github.com/chris-pearce/scally/blob/3.0.0-upgrade__functions-and-mixins/src/_settings.scss#L361).

### PR DESCRIPTION
**Pull request by chris-pearce**
Fixes #88.

### CODE REVIEW

File: src/_mixins-functions.scss [hunk 1]
```diff
@@ -60,55 +60,97 @@
...
+///                (min-resolution: 192dpi) {
+///             background-image: url('arrow@2x.png');
 ///             background-size: 32px 32px;
 ///         }
 ///     }
+///     .element-3 {
+///         @media (min-resolution: 2dppx),
+///                (min-resolution: 192dpi) {
+///             background-image: url('delete@2x.png');
+///             background-size: 8.375rem 4.062rem;
```
Comment (DaveOrDead) [inline src/_mixins-functions.scss:115]: just curious why the example is using such weird numbers

**Comment by DaveOrDead** [comment]
LGTM

File: src/_mixins-functions.scss (hunk 1 above)
Comment (chris-pearce) [reply src/_mixins-functions.scss:115]: Is it the decimals in the `rem` numbers? If so yeah that's not normal, I'll change to whole numbers.

Comment (chris-pearce) [reply src/_mixins-functions.scss:115]: I see what's happening, so the actual implementation does use whole numbers (see the `@example scss - Usage` section above):

```scss
@include hidpi-bg-img('delete@2x.png', rem(134), rem(65));
```

This is the output:

```css
background-size: 8.375rem 4.062rem;
```

**Pull request merged**